cluster_tools/copy_volume/copy_volume.py

```diff
@@ -114,8 +115,14 @@ def run_impl(self):
             out_shape = out_shape[1:]
 
         compression = task_config.pop("compression", "gzip")
+
         dtype = str(ds_dtype) if self.dtype is None else self.dtype
-        dtype = DTYPE_MAPPING.get(dtype, dtype)
+
+        dtype = DTYPE_MAPPING.get(self.dtype, self.dtype)
```

Comment:
```suggestion
        dtype = DTYPE_MAPPING.get(dtype, dtype)
```

Reply:
No, this has to be `dtype` everywhere. `self.dtype` is usually `None` and if it's not None `dtype` will be set to `self.dtype` in line 119.